src/natcap/invest/delineateit/delineateit.py

```diff
@@ -44,9 +46,17 @@
                 "each cell."),
             "name": "Digital Elevation Model"
         },
+        "detect_pour_points": {
+            "type": "boolean",
+            "required": False,
+            "about": (
+                "If ``True``, the pour point detection algorithm will run, "
+                "creating a point vector file pour_points.gpkg."),
+            "name": "Detect pour points"
+        },
         "outlet_vector_path": {
             "type": "vector",
-            "required": True,
+            "required": "not detect_pour_points",
```

Comment:
I see that this is define here, but I'm somehow seeing a validation error when the "outlet vector path" is undefined and the "detect pour points" box is checked.

Reply:
I think this was fixed by adding it to `self.assemble_args`